recipe/meta.yaml

```diff
@@ -1,16 +1,15 @@
 {% set version = "3.3.10" %}
-{% set build = 3 %}
+{% set build = 4 %}
 
 
 # ensure mpi is defined (needed for conda-smithy recipe-lint)
 {% set mpi = mpi or 'nompi' %}
 
 package:
-  name: fftw
+  name: fftw-split
```

Comment:
Does it have to be a different name? I'm not sure if it matters, but just wondering whether changing the name will clash with any bot automation.

Reply:
There is some inconsistent behavior in how conda treats split packages where the top level package has the same name as one of the outputs.

I guess caution has made people just call it something else.